### PR TITLE
Create event log screen

### PR DESCRIPTION
**Pull request by WojcikMike**
Two screenshots are screaming for Sergios Attention:
Link on the dashboard:
image
And event page:
image

At the moment there is no other way to get to that page than through the link, I think it should be available from the menu, but I have no good idea where should it reside.

### CODE REVIEW

File: src/ServicePulse.Host/app/modules/shell/js/events.route.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿(function (window, angular) {
+    
+    'use strict';
+
+    function routeProvider($routeProvider) {
+        const template = require('./../views/events-view.html');
+        $routeProvider.when('/events', {
+            data: {
+                pageTitle: "Events"
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/modules/shell/js/events.route.js:9]:
```suggestion
                pageTitle: 'Events'
```

File: src/ServicePulse.Host/app/modules/shell/js/services/service.auditLog.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+(function(window, angular, $) {
+    'use strict';
+
+    function Service($http, uri, Rx, connectionsManager) {
+
+        var scu = connectionsManager.getServiceControlUrl();
+        
+        function getEventLogItems(pageNo, pageSize) {
+            return $http.get(uri.join(scu, 'eventlogitems') + "?page=" + pageNo + "&per_page=" + pageSize)
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/modules/shell/js/services/service.auditLog.js:9]:
```suggestion
            return $http.get(uri.join(scu, 'eventlogitems') + '?page=' + pageNo + '&per_page=' + pageSize)
```

**Comment by WojcikMike** [comment]
image
The icons used are above, Book for the menu, exclamation mark for the integration errors

**Comment by WojcikMike** [comment]
rebased, ready to be reviewed

**Pull request merged**